README.md

```diff
@@ -61,46 +61,51 @@ There are two networks available:
 
 While running Ansible playbook, `network_version` needs to be set.
 
+Similarly for `heimdall_network` following options can be used
+
+* `mainnet` (Mainnet v1)
+* `mumbai`  (Mumbai testnet)
+
 ### Sentry node setup
 
 To show list of hosts where the playbook will run (notice `--list-hosts` at the end):
 
 ```bash
-ansible-playbook -l sentry playbooks/network.yml --extra-var="bor_branch=v0.2.1 heimdall_branch=v0.2.0-mainnet-1d8aca37 network_version=mainnet-v1 node_type=sentry/sentry" --list-hosts
+ansible-playbook -l sentry playbooks/network.yml --extra-var="bor_branch=v0.2.1 heimdall_branch=v0.2.0-mainnet-1d8aca37 network_version=mainnet-v1 node_type=sentry/sentry heimdall_network=mainnet" --list-hosts
```

Comment:
Let's update these tags to the latest version.

Reply:
Sure

Reply:
Updated